src/improvements/tasks/eth/006-gas-params-op/README.md

```diff
@@ -0,0 +1,42 @@
+# 006-gas-params-op: (Ethereum) Increase gas target : OP Mainnet
+
+Status: TODO
+
+## Objective
+
+Sets the following gas params in the SystemConfig contract for OP Mainnet:
```

Comment:
Is there a gov post for this task? Lets link it if there is.

Reply:
Not that I'm aware of. What's the best way to check?

Reply:
I don't think there is. Would normally be on the governance forum. I think we only need that if the security council is signing.